### ISSUE TITLE
Make it possible to specify OLLAMA_HOST url from the command line

### ISSUE THREAD

**Issue by yanniedog** [yanniedog/crewai-autocrew#18]

**Comment by yanniedog** [comment]
I might also add an option to add ollama_host from within the menu too

**Comment by Gerkinfeltser** [comment]
Yep, this is a good idea. Quick and dirty hack I did was add `ollama_url="https://fake-ngrok-address.app" #"http://localhost:11434"` to the `ollama.py` file & replaced all instances of `url = f"{ollama_url}/what/ever"` with `url = f"{ollama_url}/what/ever"` FWIW
